client/src/visualiser-src/graph-visualiser/animation-producer/GraphAddEdgeAnimationProducer.ts

```diff
@@ -0,0 +1,3 @@
+import GraphAnimationProducer from './GraphAnimationProducer';
```

Comment:
This class is unnecessary since we're using d3. We can just return GraphAnimationProducer in the addEdge method.

Reply:
Removed.